x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/cat/RestCatTrainedModelsAction.java

```diff
@@ -272,17 +307,16 @@ private Table buildTable(
             table.addCell(config.getDescription());
             table.addCell(config.getModelType());
 
-            GetTrainedModelsStatsAction.Response.TrainedModelStats modelStats = statsMap.get(config.getModelId());
-            table.addCell(modelStats.getPipelineCount());
-            boolean hasIngestStats = modelStats != null && modelStats.getIngestStats() != null;
-            table.addCell(hasIngestStats ? modelStats.getIngestStats().totalStats().ingestCount() : 0);
-            table.addCell(
-                hasIngestStats
-                    ? TimeValue.timeValueMillis(modelStats.getIngestStats().totalStats().ingestTimeInMillis())
-                    : TimeValue.timeValueMillis(0)
+            AccumulatedStats accumulatedStats = Objects.requireNonNullElse(
+                accumulatedStatsMap.get(config.getModelId()),
+                AccumulatedStats.EMPTY
```

Comment:
Instead of all the `null` checks we'll add an "empty" stats which has all zeros and if we can't find the model id it'll use that.